operator/src/test/java/org/keycloak/operator/ClusteringE2EIT.java

```diff
@@ -136,23 +136,25 @@ public void testKeycloakCacheIsConnected() throws Exception {
                         .pods()
                         .inNamespace(namespace)
                         .withName(pod.getMetadata().getName())
-                        .portForward(8080, 8080)) {
+                        .portForward(8443, 8443)) {
```

Comment:
unfortunately we cannot get rid of the port-forwarding thing, it still doesn't work even with strict hostname.